exercises/word-count/pubspec.yaml

```diff
@@ -0,0 +1,3 @@
+name: 'word_count'
+dev_dependencies:
+  test: "^0.12.24+8"
```

Comment:
Can you change the test version to <0.13.0 to be consistent with the other exercises?